clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java

```diff
@@ -35,7 +35,7 @@
 
     private final Map<String, KafkaFuture<TopicMetadataAndConfig>> futures;
 
-    CreateTopicsResult(Map<String, KafkaFuture<TopicMetadataAndConfig>> futures) {
+    public CreateTopicsResult(Map<String, KafkaFuture<TopicMetadataAndConfig>> futures) {
```

Comment:
I'd have to make these constructors public in order to mock the response in the unit test

Reply:
What about making it protected instead and mock a sub-class? `CreateTopicsResults` is part of the public API and we might not want to make them public (would this be a public API change?)

Reply:
Yeah that's a good idea! Let me try that out.